### DISCUSSION TITLE
if I disable lsposed from orange fox's magisk manager will all the modules activated inside lsposed also be disabled?

### DISCUSSION THREAD

**Discussion by sudols** [LSPosed/LSPosed#1521] [Q&A] [answered]
example: I've xprivacylua and chimi enabled if I disable/turnoff "Riru" and "Riru - LSposed" or only "Riru" from orange fox's magisk manager will it also disable the xprivacylua and chimi modules without any problems?

**Comment by yujincheng08** [comment, marked as answer]
no. but lsposed will be disabled so no xposed module will be loaded.

**Reply by sudols** [reply]
means xprivacylua and chimi (from above example) will not be loaded on next boot?

**Reply by yujincheng08** [reply]
y

**Reply by sudols** [reply]
because lsposed is disabled means these two modules can't load right?
